charts/velero/README.md

```diff
@@ -75,6 +75,10 @@ helm upgrade <RELEASE NAME> --set initContainers.backupStorageLocation.name=aws,
 
 ## Upgrading
 
+### Upgrading to v1.3.0
+
+The [instructions found here](https://velero.io/docs/v1.3.0/upgrade-to-1.3/) will assist you in upgrading from version v1.0.0 or v1.1.0 or v1.2.0 to v1.3.0.
```

Comment:
I think the upgrade documentation for v1.3.0 only goes from v1.2.0 to v1.3.0. We haven't tested v1.0.0 -> v1.3.0 or v1.1.0 -> v1.3.0

Reply:
Updated!